int16[5] axes
int16[12] buttons
